Detector/DetFCChhECalSimple/CMakeLists.txt

```diff
@@ -14,7 +14,7 @@ find_package(ROOT COMPONENTS MathCore GenVector Geom REQUIRED)
 
 install(DIRECTORY ${CMAKE_CURRENT_LIST_DIR}/compact DESTINATION Detector/DetFCChhECalSimple)
 
-include_directories(SYSTEM ${DD4hep_INCLUDE_DIRS} ${ROOT_INCLUDE_DIRS})
+#include_directories(SYSTEM ${DD4hep_INCLUDE_DIRS} ${ROOT_INCLUDE_DIRS})
```

Comment:
Why here DD4hep and root do not need to be included anymore?

Reply:
They are included at l. 21, this was a workaround for a bug that should be fixed now.